src/go/k8s/VERSION

```diff
@@ -0,0 +1 @@
+v0.0.1
```

Comment:
Alright, so basically every time we bump this number, we release. Yep, sounds good to me 👍

Reply:
We should remove this as far as I understand our approach.